### PR TITLE
[CARBONDATA-649] fix for update with rand

### PR DESCRIPTION
**Pull request by ashwini-krishnakumar**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[CARBONDATA-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).
 - [ ] Testing done
 
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - What manual testing you have done?
        - Any additional information to help reviewers in testing this change.
         
 - [ ] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA. 
                 
---

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by ashwini-krishnakumar** [comment]
retest this please

File: examples/spark2/src/main/resources/2000_UniqData_2.csv [hunk 1]
```diff
@@ -0,0 +1,4 @@
+9000,CUST_NAME_00001,12345678901.1234
+9001,1CUST_NAME_00000,12345678901.1234
```
Comment (chenliang613) [inline examples/spark2/src/main/resources/2000_UniqData_2.csv]: Why added test file to example module ?

File: integration/spark-common-test/src/test/resources/IUD/2000_UniqData_2.csv [hunk 2]
```diff
@@ -0,0 +1,5 @@
+CUST_ID,CUST_NAME,DECIMAL_COLUMN1
```
Comment (chenliang613) [inline integration/spark-common-test/src/test/resources/IUD/2000_UniqData_2.csv]: can you simulate the data for test, don't suggest adding many csv files to repo.

**Pull request closed without merging**